src/components/Models/Model/Model.component.tsx

```diff
@@ -61,7 +60,8 @@ export const Model: FC<ModelProps> = ({
     [isTouching, touchEvent, scene]
   );
 
-  normaliseMaterialsConfig(materials, bloom, materialConfig);
+  normaliseMaterialsConfig(scene, bloom, materialConfig);
```

Comment:
This is the only place where all components will be effected, however the function traverseMaterials in Models.service will do the exact same thing. Want to avoid useGraph if possible for Nova.